qa/rspec/cli/cli_tester.rb

```diff
@@ -0,0 +1,46 @@
+# encoding: utf-8
+require_relative "../../vagrant-helpers"
+require "logstash/devutils/rake" # provide download helper
+require "stud/temporary"
+
+module CliTester
+  DOWNLOAD_DIRECTORY = File.join(File.dirname(__FILE__), "..", "..", "vendor", "download")
+
+  def run_command(cmd, host)
+    hosts = (host.nil? ? servers : Array(host))
```

Comment:
The code in the feature branch include a base class for commands, see https://github.com/elastic/logstash/blob/feature/acceptance_test/qa/rspec/commands/base.rb I see part of this code belonging there as is more like a generic.